api-tests/specs/assign-roles/assign-roles.js

```diff
@@ -0,0 +1,44 @@
+const requests = require('../../data/assign-roles/assign-roles')
+const conf = require('../../config/conf')
+const sitesEndpointUri = ':8083/sites';
+const permissionsEndpointUri = ':8082/roles';
```

Comment:
Can you rename this to be "roles" endpoint?
Permissions are something different and might be confusing here.